src/IO/Progress.h

```diff
@@ -58,9 +60,10 @@ struct ResultProgress
 {
     UInt64 result_rows = 0;
     UInt64 result_bytes = 0;
+    Int64 memory_usage = 0;
 
-    ResultProgress(UInt64 result_rows_, UInt64 result_bytes_)
-        : result_rows(result_rows_), result_bytes(result_bytes_) {}
+    ResultProgress(UInt64 result_rows_, UInt64 result_bytes_, Int64 memory_usage_ = 0)
```

Comment:
I have checked all the call sites. I think we can drop the default value for `memory_usage_`.
```suggestion
    ResultProgress(UInt64 result_rows_, UInt64 result_bytes_, Int64 memory_usage_)
```

Reply:
Done